package/freescale-imx/Config.in

```diff
@@ -109,6 +109,12 @@ if BR2_PACKAGE_FREESCALE_IMX_HAS_VIV_GPU
 source "package/freescale-imx/imx-gpu-g2d/Config.in"
 source "package/freescale-imx/imx-gpu-viv/Config.in"
 source "package/freescale-imx/kernel-module-imx-gpu-viv/Config.in"
+
+source "package/freescale-imx/imx-gstreamer/Config.in"
+source "package/freescale-imx/imx-gst-plugins-base/Config.in"
+source "package/freescale-imx/imx-gst-plugins-bad/Config.in"
+source "package/freescale-imx/imx-gst-plugins-good/Config.in"
+source "package/freescale-imx/imx-gst1.0-plugin/Config.in"
```

Comment:
You could possibly skip imx-gst1.0-plugin as this seems obsolete.

Reply:
by open source terms - maybe - but, that is the actual fork for the yocto builds on the imx Gstreamer. This is the gift that NXP delivered is a forked tree with patches in the "base", the "bad", and a few other plugins.

The open source driver that is normally used - is currently an imx.6 only driver (else just software on the imx8) - so, given the situation - it was basically just port the yocto setup into the tree for the time being.